eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -187,6 +187,33 @@ function New-GitHubIssue {
     -MaximumRetryCount 3
 }
 
+function Get-GitHubIssues {
```

Comment:
Do you use this API to get all issues with particular labels?

If so, can we have the right API name to indicate it is about labels?

Reply:
Could you also add comments on what is the format for multi-labels?

Reply:
I actually prefer to have this as a single API that can support different level of filtering. Adding label to name will make it more specific than a general API. Currently implementation is based on label and creator but this can be enhanced to make them optional and support more query options.

Reply:
If support multi filters in this API, can we make labels as optional as well? Otherwise, people has no way to use this for authors only or for assignees only. It is currently working so specific for particular purpose.